src/patterns/dosdp-patterns/boneLengthAlongAxis.yaml

```diff
@@ -0,0 +1,74 @@
+pattern_name: boneLengthAlongAxis
+
+pattern_iri: http://purl.obolibrary.org/obo/oba/patterns/boneLengthAlongAxis.yaml
+
+description: "A bone length trait that is the line along an axis of a bone."
+
+#  examples:
```

Comment:
@megbalk , could you provide some examples for this pattern that we could use here?